src/fast_check/transform.rs

```diff
@@ -0,0 +1,1481 @@
+            accessibility: Some(Accessibility::Private),
+            is_abstract: n.is_abstract,
+            is_optional: n.is_optional,
+            is_override: n.is_override,
+            readonly: false,
+            declare: false,
+            definite: !n.is_optional && !n.is_static,
+          });
+          return Ok(true);
+        }
+        let is_overload = self.public_ranges.is_impl_with_overloads(&n.range());
+        self.transform_fn(
+          &mut n.function,
+          Some(n.key.range()),
+          is_overload,
+          n.kind == MethodKind::Setter,
+        )?;
+        Ok(true)
+      }
+      ClassMember::ClassProp(n) => {
+        if n.accessibility == Some(Accessibility::Private) {
+          n.type_ann = Some(unknown_type_ann());
+          if !n.is_optional && !n.is_static {
+            n.definite = true;
+          }
+          return Ok(true);
+        }
+        if n.type_ann.is_none() {
+          let inferred_type = n
+            .value
+            .as_ref()
+            .and_then(|e| self.maybe_infer_type_from_expr(e));
+          match inferred_type {
+            Some(t) => {
+              n.type_ann = Some(Box::new(TsTypeAnn {
+                span: DUMMY_SP,
+                type_ann: Box::new(t),
+              }));
+            }
+            None => {
+              self.mark_diagnostic(
+                FastCheckDiagnostic::MissingExplicitType {
+                  range: self.source_range_to_range(n.key.range()),
+                },
+              )?;
+            }
+          }
+        }
+        n.definite = !n.is_optional && !n.is_static && !n.declare;
+        n.decorators.clear();
+        n.value = None;
+        Ok(true)
+      }
+      ClassMember::AutoAccessor(_n) => {
+        // waiting on https://github.com/swc-project/swc/pull/8436
+        // if n.accessibility == Some(Accessibility::Private) {
+        //   n.type_ann = Some(unknown_type_ann());
+        //   n.definite = true;
+        //   return Ok(true);
+        // }
+        // if n.type_ann.is_none() {
+        //   let inferred_type = n
+        //     .value
+        //     .as_ref()
+        //     .and_then(|e| self.maybe_infer_type_from_expr(&*e));
+        //   match inferred_type {
+        //     Some(t) => {
+        //       n.type_ann = Some(Box::new(TsTypeAnn {
+        //         span: DUMMY_SP,
+        //         type_ann: Box::new(t),
+        //       }));
+        //     }
+        //     None => {
+        //       self.mark_diagnostic(
+        //         FastCheckTransformDiagnostic::MissingExplicitType {
+        //           range: self.source_range_to_range(n.key.range()),
+        //         },
+        //       )?;
+        //     }
+        //   }
+        // }
+        // n.definite = true;
+        // n.decorators.clear();
+        // n.value = None;
+        // Ok(true)
+        todo!("Remove auto-accessor for now. Waiting on https://github.com/swc-project/swc/pull/8436")
```

Comment:
I'm waiting on this PR for this. I think this is ok to do for now because barely anyone uses accessors.